bin/generate-zbm

```diff
@@ -116,14 +116,16 @@ if ( ( defined $runConf{kernel} ) and ( length $runConf{kernel} ) ) {
 $runConf{bootdir} = dirname( $runConf{kernel} );
 ( $runConf{kernel_prefix}, $runConf{kernel_version} ) = split( '-', basename( $runConf{kernel} ) );
 
-# We always need to create an initramfs
-printf "Creating ZFS Boot Menu %s, with %s %s\n", $runConf{version}, $runConf{kernel_prefix}, $runConf{kernel_version};
+# We need to create an initramfs for all cases other than unifiedEFI
+if ( defined( $config{Components}{Copies} ) and ( $config{Components}{Copies} gt 0 ) ) {
```

Comment:
If you add another type of output (besides component and unified), we'd need to add another case here. Do you think there's another way to write this conditional that is more general?

Reply:
In the event that another output type is added, I can see if there's a nicer way to write that out. For now though it's reasonably easy to read so I'm inclined to leave it as-is.